src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Home.css";
+
+const Home = () => {
+    return (
+        <div className="flex jstfy-ctnt-center">
+            <div className="home-main-container grid jstfy-ctnt-spc-around align-itm-center">
+                <div className="txt-container grid">
+                    <h1>Cloud Note</h1>
+                    <h3>Meet Your Modern Note Taking App</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem vero atque porro sapiente eum corrupti
+                        consequatur culpa soluta, optio, explicabo id nostrum ad quasi aut, dicta voluptas adipisci voluptatem
+                        doloremque!
+                    </p>
+                    <Link to="/Notes">
+                        <button>Join Now</button>
```

Comment:
Suggetion : one suggestion, give primary color to your CTA button.

Reply:
Thanks for your suggestion but I like the default color and style of the button for this page.